i-Code-Doc/core/models/udop_dual.py

```diff
@@ -257,18 +249,13 @@ def forward(
                 layer_outputs = layer_outputs[:1] + (None,) + layer_outputs[1:]
             hidden_states, present_key_value_state = layer_outputs[:2]
             # We share the position biases between the layers - the first layer store them
-            # layer_outputs = hidden-states, key-value-states (self-attention weights),
-            # (self-attention position bias), (cross-attention weights), (cross-attention position bias)
+
             #import pdb; pdb.set_trace()
```

Comment:
Pls double check all file and ensure #import pdb; pdb.set_trace() are all deleted.